src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -222,14 +222,27 @@ public static Date convertLocalDateToUtc(Date localDate, double localTimeZone) {
      * Converts the {@code localDateTime} to {@code Instant} using the {@code timeZone}.
      */
     public static Instant convertLocalDateTimeToInstant(LocalDateTime localDateTime, ZoneId timeZone) {
```

Comment:
Why is it necessary to change this function as well? Is there anywhere in the code that use this function and is supposed to handle special time?

Reply:
Oops I meant the function below, not this one.

Reply:
It's more to make it the inverse of `convertLocalDateTimeToInstant`.

Reply:
Are these 2 functions used anywhere else and do they also need to handle the special time there? I was thinking of an alternative which is to write new functions with this special behavior and keep these 2 functions for purely doing timezone conversion.

Reply:
Seems they are not. Yes, I guess your alternative is better.